utilities/src/main/java/io/syndesis/qe/utils/SQSUtils.java

```diff
@@ -0,0 +1,147 @@
+package io.syndesis.qe.utils;
+
+import io.syndesis.qe.accounts.Account;
+import io.syndesis.qe.accounts.AccountsDirectory;
+
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import lombok.extern.slf4j.Slf4j;
+import software.amazon.awssdk.auth.credentials.AwsBasicCredentials;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.sqs.SqsClient;
+import software.amazon.awssdk.services.sqs.model.Message;
+import software.amazon.awssdk.services.sqs.model.QueueAttributeName;
+import software.amazon.awssdk.services.sqs.model.QueueDoesNotExistException;
+import software.amazon.awssdk.services.sqs.model.SendMessageBatchRequest;
+import software.amazon.awssdk.services.sqs.model.SendMessageBatchRequestEntry;
+
+@Slf4j
+@Component
+@Lazy
+public class SQSUtils {
+    private static final String AWS_ACCOUNT_ID = "928544467084";
```

Comment:
Shouldn't this also be in credentials.json?

Reply:
yup, I already sent a PR to ci-ops-fuse

Reply:
So you'll remove it from here?

Reply:
yes yes yes